guides/common/modules/proc_generating-host-monitoring-reports.adoc

```diff
@@ -31,3 +31,26 @@ If you have selected the *Send report via e-mail* checkbox, the host monitoring
 +
 This command waits until the report fully generates before completing.
 If you want to generate the report as a background task, you can use the `hammer report-template schedule` command.
++
+[NOTE]
+====
+If you want to generate a subscription entitlement report, you have to use the `Days from Now` option to specify the latest expiration time of entitlement subscriptions.
+You can use the `no limit` value to show all entitlements.
+See the examples below.
```

Comment:
Please reword this sentence.

Reply:
Please check the new format to see if it's better.